tests/e2e/utils/mock-requests.js

```diff
@@ -129,7 +148,6 @@ export default class MockRequests {
 	 * Fulfill the MC connection request.
 	 *
 	 * @param {Object} payload
-	 * @return {Promise<void>}
```

Comment:
I don't think this return docs should be removed here or for `fulfillAdsConnection`